.goreleaser.yml

```diff
@@ -26,31 +23,41 @@ builds:
     ignore:
       - goos: darwin
         goarch: '386'
-    binary: '{{ .ProjectName }}_v{{ .Version }}'
+    env:
+      - CGO_ENABLED=0
+    flags:
+      - -trimpath
+    ldflags:
+      - '-s -w -X main.version={{.Version}} -X main.commit={{.Commit}}'
```

Comment:
This looks to be invalid for this repo and we'd like to keep it the way it works now

Reply:
Which portion?